src/Rakaposhi.Core/Rakaposhi.Business.Core/DBRepository/DBRepositoryFactory.cs

```diff
@@ -24,13 +25,27 @@ public IUserStatusRepository UserStatusRepository
         {
             get
             {
-                if( _userStatusRepository is null)
+                if (_userStatusRepository is null)
```

Comment:
What I feel, It is not a change of this PR.
Please Remove space between if and its brackets.

Reply:
fixed